src/swtpm/seccomp_profile.c

```diff
@@ -102,7 +102,7 @@ int create_seccomp_profile(bool cusetpm, unsigned int action)
         SCMP_SYS(fsconfig),
 #endif
 #ifdef __NR_fsmount
-        SCMP_SYS(fs_mount),
+        SCMP_SYS(fsmount),
```

Comment:
Most of the commit text can go. Just leave this part:
```
The project wouldn't compile on my ubuntu 20.04.1 based system with the error message:
```
  CC       libswtpm_libtpms_la-seccomp_profile.lo
In file included from seccomp_profile.c:44:
seccomp_profile.c: In function ‘create_seccomp_profile’:
seccomp_profile.c:105:9: error: ‘__SNR_fs_mount’ undeclared (first use in this function)
  105 |         SCMP_SYS(fs_mount),
      |         ^~~~~~~~
seccomp_profile.c:105:9: note: each undeclared identifier is reported only once for each function it appears in

Signed-off-by: Hans Niklas Jacob <hnj@posteo.de>
```